packages/dropdownable/components/Dropdownable.tsx

```diff
@@ -90,7 +75,7 @@ const Dropdownable: React.FC<DropdownableProps> = ({
     React.useState<HTMLDivElement | null>(null);
   const { styles, attributes } = usePopper(referenceElement, popperElement, {
     placement: getPreferredDirection(preferredDirections),
-    modifiers: [maxSize, applyMaxSize, getFlipModifier(preferredDirections)]
```

Comment:
I think this is just deprecating functionality which means it would be a BREAKING CHANGE.

Reply:
I fixed it in another way now.